pepper-apis/src/main/java/org/broadinstitute/ddp/json/activity/ActivityInstanceSummary.java

```diff
@@ -11,20 +11,22 @@
     @SerializedName("instanceGuid")
     private String activityInstanceGuid;
 
-    @SerializedName("activityName")
+    // todo: rename to `activityName`
+    @SerializedName("activityDashboardName")
```

Comment:
Changing it in this PR will change the API. I'll do the change in a separate PR instead.